Chapter_9/Ex9.rb

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env ruby
+
+# Here's some new strange stuff, remember type it exactly.
+
+days = 'Mon Tue Wed Thu Fri Sat Sun'
+months = "Jan\nFeb\nMar\nApr\nMay\nJun\nJul\nAug"
+
+puts "Here are the days: #{days}"
+puts "Here are the months: #{months}"
+
+puts "There's something going on here.\nWith the three double-quotes."
```

Comment:
Try doing this without three puts statements. You can either use a HEREDOC, or try doing the same thing he wrote down, but instead using single quotes. Does it still work?